graphql/ui_plugin_build_baron.go

```diff
@@ -54,7 +54,9 @@ func BbFileTicket(context context.Context, taskId string, execution int) (bool,
 	}
 
 	webHook, _ := plugin.IsWebhookConfigured(t.Project, t.Version)
-	if webHook.Endpoint != "" {
+	if webHook.Endpoint != "" && bbProject.TicketCreateProject != "" {
```

Comment:
I think I would prefer for this check to be in IsWebhookConfigured so that it covers more use cases.